### DISCUSSION TITLE
Unraid CA Store

### DISCUSSION THREAD

**Discussion by SavageAUS** [enchant97/web-portal#40] [General]
Do you mind if I put this on the unraid community applications store?
Do you have an icon for the app?

**Comment by SavageAUS** [comment]
I found an icon to use. 
https://raw.githubusercontent.com/enchant97/web-portal/main/web_portal/static/favicon.ico

**Reply by SavageAUS** [reply]
![image](https://user-images.githubusercontent.com/17326832/185771241-e890eddd-f054-4681-a738-eb1c2e6ca0f6.jpeg)
This is what it would look like.

**Reply by enchant97** [reply]
Sounds like a great idea! Only requirement (apart from following the AGPL license of course) would be to say that it's unofficial, as I have no way of testing and therefore wont be able to provide support for that particular platform.

As for icons I don't know what format is required, but I have a png and svg version. They will be higher quality than the ico file:

- <https://raw.githubusercontent.com/enchant97/web-portal/main/web_portal/static/icon-256.png>
- <https://raw.githubusercontent.com/enchant97/web-portal/main/web_portal/static/icon.svg>

> Side note: I also copy all my app icons to a central repository at <https://github.com/enchant97/app-icons>

**Reply by SavageAUS** [reply]
Not sure if you are familiar with the unraid ca store but for each "app" links to the pjs t can and will be provided so that should cover the licensing. I can also add a note into the description saying that it is unofficial (as most of the ca store is). I will also send you the xml to look over before I send to ca store admin.

**Comment by SavageAUS** [comment]
Finally got around to creating the template.
rename to web-portal.xml
[web-portal.xml.txt](https://github.com/enchant97/web-portal/files/9437793/web-portal.xml.txt)

**Reply by SavageAUS** [reply]
Can you confirm that the template looks ok and that the variable for uploading icons is fine as i cannot see where to upload icons.

**Reply by enchant97** [reply]
Looks ok, I've never used unraid so I can't say whether it's a valid config.

As an aside, I would like to warn that in the future the project website will be moved to a separate subdomain (docs.enchantedcode.co.uk/web-portal). So this will need to be updated for your config when it moves.

**Reply by SavageAUS** [reply]
Yeah no worries. I will monitor it and if I need to change anything I will. 
I figured out the icons too. 
I'm just waiting for an unraid moderator to let me know if the xml is good or not. 
It's just docker, basically. Mounting host path to container etc... done in xml instead of a docker-compose. 
I think it's a great app and more people should see it.

**Reply by enchant97** [reply]
Thanks. Do you have any plugin ideas or other suggestions for the app?
